pkg/dbs/userdb/db_methods.go

```diff
@@ -172,6 +173,10 @@ func (dbService *UserDBService) UpdateLoginTime(instanceID string, id string) er
 	if err != nil {
 		return err
 	}
+	_, err = dbService.UpdateMarkedForDeletionTime(instanceID, id, 0, true)
```

Comment:
I think, it might be better not to call DB methods twice, but do one update for all attributes at once.
Instead of:
`update := bson.M{"$set": bson.M{"timestamps.lastLogin": time.Now().Unix()}}`
you can update both attributes, like:
`update := bson.M{"$set": bson.M{"contactPreferences": prefs, "timestamps.updatedAt": time.Now().Unix()}}`
(just example not exact code)

Reply:
It is now one update for all attributes. Please check.